internal/v1/api.yaml

```diff
@@ -1069,6 +1069,14 @@ components:
           $ref: '#/components/schemas/Customizations'
         metadata:
           $ref: '#/components/schemas/BlueprintMetadata'
+        custom_repositories_details:
```

Comment:
One observation - this export is closer to console dot than the IB blueprint, this could be named content_sources or something like that.